src/api/customer/customer-validator.ts

```diff
@@ -34,6 +34,31 @@ const createCustomerValidator = () => {
     });
 };
 
+const findAllCustomerValidator = () => {
+   return celebrate({
+       [Segments.QUERY]: Joi.object().keys({
+           offset: Joi
+           .number(),
+           limit: Joi
+           .number()
+           .min(1)
+       })
+   });
```

Comment:
Já poderia ter especificado valores defaults caso o usuário não informasse nada.
E também que não aceita números fracionados (ex `1.52`, mas apenas inteiros.

```suggestion
       [Segments.QUERY]: Joi.object().keys({
           offset: Joi
             .number()
             .integer()
             .min(0)
             .default(0),
           limit: Joi
             .number()
             .integer()
             .min(1)
             .max(50)
             .default(50)
       })
   });
```